src/plugins/Library/Library.java

```diff
@@ -293,7 +293,7 @@ public synchronized void saveState(){
 						if (!(ce instanceof ExpectedMIMEEvent)) { return; }
 						synchronized(c) {
 							String type = ((ExpectedMIMEEvent)ce).expectedMIMEType;
-							System.out.println("Expected type in index: "+type);
+							Logger.error(this, "Expected type in index: "+type);
```

Comment:
This doesn’t really seem to be an error condition, does it have to be logged as error?